scripts/inference.py

```diff
@@ -202,11 +213,19 @@ def ids_for_prompt(prompt):
 def print_result(result):
     if local_rank != 0:
         return
-    # stop at EOS token if present
+    if padding_kwargs is not None:
```

Comment:
why are we running `trim_prefix` twice? Do we pad 0s before `bos_token_id`?

Reply:
yes